docs/embedded/development/content-experiences/office-experience.md

```diff
@@ -21,6 +21,11 @@ Versioning is automatically enabled on each Word, Excel and PowerPoint file stor
 
 It's simple for your users to collaborate on your SharePoint Embedded Application's Office documents – they can **Share documents** with specific peers or with people outside your organization by Creating a shareable link to use wherever needed, Send an email invitation or @mention in comments to tag someone for feedback and, **Collaborate in real time** by co-authoring in Office with SharePoint Embedded Applications.
 
+> [!NOTE]
+> Mentions require target users to [have an M365 license assigned to them](../auth.md#mention-users-in-office-documents).
+>
+> Mentions are restricted to people inside the consuming tenant's organization. Mentions excludes Guests and users from other tenants in a Multi-Tenant setting.
```

Comment:
This was copied directly from the @-mentions documentation, but having "Guests" (upper-case G) and "Multi-Tenant" (upper-case M and T) looks a bit off. I'll leave this up for reviewers, I'd change all of those to lower-case.

Reply:
I suggest, not just for this, but for *all* content updates, is to run this through Acrolinx. You can learn more about this from the MSDocs/MSLearn MSFT-internal site.

In a nutshell, Acrolinx is like Grammarly, but for Microsoft's style guide. MSDocs uses it, and all content published to MSLearn's training section (modules, labs, etc), must have a score of 80+ to get published. It finds things like these.

In this case, it doesn't flag the uppercase "Guest", but the correct use of the other is "multitenant" (one word, all lowercase). I'll submit my changes from running this page through Acrolinx